client-app/src/styles.css

```diff
@@ -43,3 +43,7 @@ body {
   font-size: 1.7em;
   font-weight: normal;
 }
+
+/* .react-datepicker-wrapper {
+  width: 100%;
+} */
```

Comment:
why it is commented

Reply:
in from if date picker does not expand to 100%, we can use this, just for safety